stream/BidirectionalStream.java

```diff
@@ -108,6 +111,7 @@ public void close() {
 
     private void close(@Nullable StatusRuntimeException error) {
         if (isOpen.compareAndSet(true, false)) {
+            this.error = error;
```

Comment:
store the closure error against the bidirectional stream to prevent future transaction operations